modules/core/react/stories/stories.tsx

```diff
@@ -44,62 +74,166 @@ export const type = (hierarchy: any) => (
+      <label css={type.variant.label}>Label Text</label>
+      <span css={type.variant.hint}>Hint Text</span>
+      <span css={type.variant.error}>Error Text</span>
+      <div>
+        <a href="#" css={type.variant.link}>
+          Link Text
+        </a>
+      </div>
+      <span css={[type.variant.inverse, inverseStyle]}>Inverse Text</span>
+      <span css={type.variant.mono}>Mono Text</span>
+    </div>
+  </React.Fragment>
 );
 
-storiesOf('Tokens|Core/React', module)
-  .addDecorator(withReadme(README))
-  .add('Depth', () => {
-    const Card = styled('div')({
-      width: 200,
-      height: 200,
-      margin: 20,
-      borderRadius: 4,
-      display: 'flex',
-      alignItems: 'center',
-      justifyContent: 'center',
-    });
-
-    return (
-      <div className="story">
-        <div style={{display: 'flex'}}>
-          <Card style={canvas.depth.inset}>Depth -1</Card>
-          <Card style={canvas.depth['1']}>Depth 1</Card>
-          <Card style={canvas.depth['2']}>Depth 2</Card>
-          <Card style={canvas.depth['3']}>Depth 3</Card>
-          <Card style={canvas.depth['4']}>Depth 4</Card>
-        </div>
-      </div>
-    );
+const Shape = styled('div')<{radius?: string | number; size?: string | number}>(
+  {
+    ...type.h4,
+    margin: spacing.m,
+    background: colors.blueberry400,
+    color: colors.frenchVanilla100,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flexDirection: 'column',
+    '& span': {
+      ...type.body2,
+      ...type.variant.mono,
+      display: 'block',
+      color: colors.blueberry100,
+    },
+  },
+  ({radius}) => ({
+    borderRadius: radius,
+  }),
+  ({size}) => ({
+    width: size || spacing.xxl,
+    height: size || spacing.xxl,
   })
-  .add('Type', () => {
-    return (
-      <div className="story">
-        <section>
-          <h1 style={canvas.type.dataViz1}>Data Viz 1 Header</h1>
-          <h1 style={canvas.type.dataViz2}>Data Viz 2 Header</h1>
-          <H1>Styled Component H1 Header</H1>
-          {type(canvas.type)}
-        </section>
+);
+
+const SizeLabel = styled('div')({
+  ...type.h4,
+  margin: spacing.s,
+  width: 80,
+  '& span': {
+    ...type.body2,
+    ...type.variant.mono,
+    ...type.variant.hint,
+    display: 'block',
+  },
+});
+
+export const BorderRadius = () => (
+  <React.Fragment>
+    {Object.keys(borderRadius).map(size => (
+      <div css={{display: 'flex'}}>
+        <SizeLabel>
+          {size}
+          <span>({borderRadius[size]})</span>
+        </SizeLabel>
+        <Shape radius={borderRadius[size]} />
       </div>
-    );
-  });
+    ))}
+  </React.Fragment>
+);
+
+export const Spacing = () => (
+  <React.Fragment>
+    {Object.keys(spacing).map(size => (
+      <div css={{display: 'flex'}}>
+        <SizeLabel>
+          {size}
+          <span>({(spacing as any)[size]})</span>
+        </SizeLabel>
+        <Shape size={(spacing as any)[size]} radius={borderRadius.m} />
```

Comment:
This seems to make `xxxs` and `xxs` look round and less clear on the sizing. Is there a specific reason that there's a non-zero border radius for this shape?

![image](https://user-images.githubusercontent.com/14299381/84230911-dd735080-aaa1-11ea-879d-15bfd6fd0f33.png)